src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -228,14 +228,13 @@ private void testCreateInstructorAction() throws Exception {
 
         feedbackEditPage.clickEditSessionButton();
 
-        FeedbackSessionAttributes feedbackSession = BackDoor.getFeedbackSession(demoCourseId,
-                                                                                "Second team feedback session");
+        FeedbackSessionAttributes feedbackSession = getFeedbackSession(demoCourseId, "Second team feedback session");
         feedbackEditPage.editFeedbackSession(feedbackSession.getStartTime(),
```

Comment:
According to the CI, this test still needs around 2-3 rounds to pass. There error is here.
``` shell
ci-tests > sequential-ui-tests > teammates.test.cases.browsertests.AdminHomePageUiTest.testAll FAILED
    java.lang.NullPointerException at AdminHomePageUiTest.java:232 
```
Can you examine what is happening?